wolox-books/src/app/app.component.scss

```diff
@@ -0,0 +1,60 @@
+@import '../sass/colors';
+@import '../sass/fonts';
+
+.navbar{
+  border-top: 5px solid $blue;
+  padding: 40px;
+  text-align: center;
+
+  .logo {
+    max-width: 500px;
+    display: block;
+    margin: 0 auto;
+  }
+
+  .subtitle{
+    font-weight: bold;
+    letter-spacing: 8px;
+    font-size: 1em;
+    margin-top: 10px;
+    max-width: 500px;
+  }
+}
+
+.main {
+  padding: 20px;
+  max-width: 500px;
+  margin: 0 auto;
+}
+
+.reg-form {
+
+  .prefix {
+    padding: 5px 7px;
```

Comment:
The separation should be achieved using margins. If I add a background color it would make sense to still have a gap separating it from the input